backend/src/zimfarm_backend/common/external.py

```diff
@@ -159,18 +161,6 @@ def get_statement() -> dict[str, Any]:
     )
 
 
-class ExternalIpUpdater:
```

Comment:
I'm so sorry, I just realize now why we might need this: for tests. I used this to inject a different behavior during tests. Don't know if this is feasible without this class, but I'm not sure this will work with a static method either. Anyway this way kinda messy. Let's merge this without this class, you will anyway be able to easily restore it should it prove to be mandatory for testing.